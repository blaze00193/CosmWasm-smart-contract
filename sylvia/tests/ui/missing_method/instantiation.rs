pub struct Contract {}

#[sylvia::contract]
impl Contract {
    pub const fn new() -> Self {
        Self {}
    }
}

fn main() {}
