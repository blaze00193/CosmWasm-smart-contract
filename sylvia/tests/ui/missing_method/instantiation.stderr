error: Missing instantiation message.

         = note: `sylvia::contract` requires exactly one method marked with `#[sv::msg(instantiation)]` attribute.

 --> tests/ui/missing_method/instantiation.rs:4:1
  |
4 | impl Contract {
  | ^^^^
